daemon/tests/happy_path.rs

```diff
@@ -125,6 +126,41 @@ async fn taker_takes_order_and_maker_accepts_and_contract_setup() {
     assert!(matches!(maker_cfd.state, CfdState::Open { .. }));
 }
 
+#[tokio::test]
+async fn collaboratively_close_an_open_cfd() {
+    let _guard = init_tracing();
+    let (mut maker, mut taker, order_id) = start_from_open_cfd_state().await;
+
+    taker.propose_settlement(order_id).await;
+
+    let (taker_cfd, maker_cfd) = next_cfd(taker.cfd_feed(), maker.cfd_feed()).await.unwrap();
+    assert!(matches!(
+        taker_cfd.state,
+        CfdState::OutgoingSettlementProposal { .. }
+    ));
+    assert!(matches!(
+        maker_cfd.state,
+        CfdState::IncomingSettlementProposal { .. }
+    ));
+
+    maker.mocks.mock_monitor_collaborative_settlement().await;
+    taker.mocks.mock_monitor_collaborative_settlement().await;
+
+    maker.accept_settlement_proposal(order_id).await;
+    sleep(Duration::from_secs(5)).await; // need to wait a bit until both transition
+
+    let (taker_cfd, maker_cfd) = next_cfd(taker.cfd_feed(), maker.cfd_feed()).await.unwrap();
+    assert!(matches!(taker_cfd.state, CfdState::PendingClose { .. }));
+    assert!(matches!(maker_cfd.state, CfdState::PendingClose { .. }));
```

Comment:
Can we someone assert that this is a collab. close?

Reply:
`Close` is actually a collab-close only closure! a force-close would assert on `PendingCommit`